### PR TITLE
added git-annex-remote-googledrive

### PR DESCRIPTION
**Pull request by notestaff**
Checklist

- [X] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [X] Source is from official source
- [X] Package does not vend other packages
- [X] Build number is 0
- [X] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

File: recipes/git-annex-remote-googledrive/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+test:
+  commands:
+    - git-annex-remote-googledrive version
+
+about:
+  home: "https://github.com/Lykos153/git-annex-remote-googledrive"
+  license: "GPL-3.0"
+  license_family: "GPL"
+  license_file: '{{ environ["RECIPE_DIR"] }}/LICENSE'
```
Comment (scopatz) [inline recipes/git-annex-remote-googledrive/meta.yaml]: You don't have the license file in this PR

**Comment by scopatz** [comment]
The license seem to have the wrong path: https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=44521&view=logs&jobId=10bda42a-d420-5c28-a43f-bc4e66405e5c&taskId=bef775d0-a1a2-55a3-9721-17147029baf0&lineStart=507&lineEnd=508&colStart=1&colEnd=1

**Comment by notestaff** [comment]
@scopatz thanks, should be fixed now.

File: recipes/git-annex-remote-googledrive/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
```
Comment (chrisburr) [inline recipes/git-annex-remote-googledrive/meta.yaml]: Conda will automatically look in the recipe directory if it isn't in the build directory:
```suggestion
  license_file: LICENSE
```

File: recipes/git-annex-remote-googledrive/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: "8d3edcd9cbcf7c7f5617eaacd14f275ed6b5cbafd3e66f4f3a0f75d86b675eed"
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+  skip: True  # [py2k or not linux]
```
Comment (chrisburr) [inline recipes/git-annex-remote-googledrive/meta.yaml]: Could this be `noarch` instead?
```suggestion
  noarch: python
```
(It doesn't matter if Windows or macOS then fail)

File: recipes/git-annex-remote-googledrive/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+
+requirements:
+  host:
+    - pip
+    - python
```
Comment (chrisburr) [inline recipes/git-annex-remote-googledrive/meta.yaml]: If this is made `noarch`:
```suggestion
    - python >=3
```

File: recipes/git-annex-remote-googledrive/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+  run:
+    - annexremote
+    - pydrive
+    - python
```
Comment (chrisburr) [inline recipes/git-annex-remote-googledrive/meta.yaml]: If this is made `noarch`:
```suggestion
    - python >=3
```

**Comment by notestaff** [comment]
@chrisburr made python: noarch .  The windows build is failing though, so not sure how to mark this PR as ready to be looked at again?

**Pull request merged**